spec/adapters/replication_transfer_adapter_spec.rb

```diff
@@ -17,6 +17,7 @@
   updated_at = "2015-02-25T15:27:40Z"
   created_at = "2015-02-25T15:27:40Z"
   link = "user@herp.derp.org:/blah"
+  cancel_reason_detail = "testing is fun # comment // woo"
```

Comment:
Do you mean like Fabricate(:cancel_reason_detail) ?  I'm not sure if Fabricators are built to handle plain objects, but we could certainly roll our own or just replace the string literal with an appropriate Faker call.